### PR TITLE
multiple[patch]: Switch deprecated `model.call` to `model.invoke` in tests and examples

### PR DESCRIPTION
**Pull request by janvi-kalra**
No-op. 
Noticed that `model.call` was deprecated so updating the library and documentation to `mode.invoke` 
Starter PR just to get the lay of the land 
Ran `yarn test` and all tests continued to pass.

### CODE REVIEW

File: docs/core_docs/.gitignore [hunk 1]
```diff
@@ -43,6 +43,8 @@ docs/use_cases/question_answering/sources.md
 docs/use_cases/question_answering/sources.mdx
```
Comment (janvi-kalra) [inline docs/core_docs/.gitignore:43]: seems to have happened automatically - didn't manually change this

Comment (jacoblee93) [reply docs/core_docs/.gitignore:43]: Yeah, all good there

**Comment by jacoblee93** [comment]
Fantastic, thank you!

**Comment by jacoblee93** [comment]
Thank you! And congrats on your first PR!

**Pull request merged**